connector/protobuf/src/main/scala/org/apache/spark/sql/protobuf/utils/SchemaConverters.scala

```diff
@@ -66,6 +66,10 @@ object SchemaConverters {
         Some(DayTimeIntervalType.defaultConcreteType)
       case MESSAGE if fd.getMessageType.getName == "Timestamp" =>
         Some(TimestampType)
+        // FIXME: Is the above accurate? Users can have protos named "Timestamp" but are not
```

Comment:
@rangadi How about adding a TODO comment about verifying fields (seconds, nanos) in Timestamp and Duration, and throwing an error if it has more or less? I'm adding an error class in SPARK-40777

Reply:
Yeah, we could verify it matches a message like `{ int64 seconds = 1; int32 nanos = 2 }`. This will match with ones defined in protobuf package too.